Makefile

```diff
@@ -24,7 +24,9 @@ WPTD_GO_PATH ?= $(WPT_GO_PATH)/wpt.fyi
 NODE_SELENIUM_PATH ?= $(WPTD_PATH)webapp/node_modules/selenium-standalone/.selenium/
 SELENIUM_SERVER_PATH ?= $(NODE_SELENIUM_PATH)selenium-server/3.8.1-server.jar
 GECKODRIVER_PATH ?= $(NODE_SELENIUM_PATH)geckodriver/0.20.0-x64-geckodriver
+CHROMEDRIVER_PATH ?= $(NODE_SELENIUM_PATH)chromedriver/2.37-x64-chromedriver
```

Comment:
Just to confirm: we are not downloading the binary ourselves, but instead they come with the node Selenium package, correct?

(The reason I ask is because this is not the most recent release of Chromedriver, but it's not a problem if they are provided by something indirectly.)

Reply:
Yes, we're just leveraging the install that's done by Selenium. I guess it will eventually break when the node package starts fetching a newer version, but we'll deal with that when it happens.